client/reader/following/main.tsx

```diff
@@ -47,4 +58,42 @@ function FollowingStream( { ...props } ) {
 	);
 }
 
+function ReaderStreamSidebar(): JSX.Element | null {
```

Comment:
It looks like there is a `/reader/stream` directory that is where these appear to go. Did you have specific reason for not putting this in that directory? If not, I might suggest sticking to that pattern because it makes it easier to read and find 😄

Reply:
Done in https://github.com/Automattic/wp-calypso/pull/96954/commits/ea5e8921436e53785af102b824b1b566285fbf3d.